roles/tpa_single_node/templates/manifests/vexination/walker/Deployment.yaml.j2

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vexination-walker
+  namespace: "{{ tpa_single_node_namespace }}"
+  labels:
+    name: vexination-walker
+    app: vexination-walker
+spec:
+  selector:
+    matchLabels:
+      app: vexination-walker
+  template:
+    spec:
+      restartPolicy: Never
+      volumes:
+        - name: walker-state
+          persistentVolumeClaim:
+            claimName: vexination-walker-state
+      containers:
+        - image: "{{ tpa_single_node_trustification_image }}"
+          imagePullPolicy: Always
+          name: walker
+          command:
+            - /trust
+          args:
+            - vexination
+            - walker
+            - --sink
+            - http://vexination-api-pod:8080/api/v1/vex
+            - --source
+            - https://www.redhat.com/.well-known/csaf/provider-metadata.json
+            - --ignore-distributions
+            - https://access.redhat.com/security/data/csaf/v2/advisories/
+            - '-3'
+            - --since-file
+            - /walker-state/since
+          env:
+            - name: RUST_LOG
+              value: "info"
+            - name: INFRASTRUCTURE_ENABLED
+              value: "true"
+            - name: INFRASTRUCTURE_BIND
+              value: "[::]:9010"
+            - name: OIDC_PROVIDER_CLIENT_ID
+              value: "{{ tpa_single_node_oidc_provider_client_id }}"
```

Comment:
Please use the client id which is also provided in the storage_secret.
Hiding the "username" allows to provide less "surface" attacks.

Reply:
@desmax74, update,  sorry the id is not yet in the storage_secret, but it would make sense to store it there too, or move it to a client dedicated secret manifest.